### PR TITLE
[#183] Replace UFO type labels.

### PR DESCRIPTION
**Pull request by ledsoft**
Changed content types based on https://github.com/kbss-cvut/termit-ui/issues/183

### CODE REVIEW

File: src/main/resources/languages/language.ttl [hunk 1]
```diff
@@ -55,8 +55,8 @@ ufo:object-type a a-popis-dat:term ;
 
 ufo:relator a a-popis-dat:term ;
     rdfs:label "Vztah"@cs,
-        "Relator"@en .
+        "Relation"@en .
 
 ufo:relator-type a a-popis-dat:term ;
     rdfs:label "Typ vztahu"@cs,
-        "Relator Type"@en .
+        "Relation Type"@en .
```
Comment (psiotwo) [inline src/main/resources/languages/language.ttl]: Relator

**Review by psiotwo — DISMISSED** [review]
see comment, otherwise ok

File: src/main/resources/languages/language.ttl (hunk 1 above)
Comment (ledsoft) [reply src/main/resources/languages/language.ttl]: So it's "Relation" but "Relator Type"?

Comment (psiotwo) [reply src/main/resources/languages/language.ttl]: No, I meant Relator, not 'Relator type'. 

But to be sure I looked again in [doc](https://dev.nemo.inf.ufes.br/seon/UFO.html) and I was wrong, it is vice versa, so:

Relation Type -> Relator
Relator Type -> Relation

Comment (ledsoft) [reply src/main/resources/languages/language.ttl]: I still don't get it. The language (as can be seen in the file) contains only "Relator" and "Relator type". So how do we change their translation?

Comment (psiotwo) [reply src/main/resources/languages/language.ttl]: see my commit

**Pull request merged**